test/test_helper.rb

```diff
@@ -15,3 +15,10 @@
 require 'babelish'
 
 require "babelish/commandline"
+
+require 'webmock/test_unit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.